core/templates/dev/head/admin/admin.html

```diff
@@ -108,118 +208,148 @@
+      <md-card class="oppia-page-card oppia-long-text" ng-if="currentTab === 'TAB_JOBS'" style="max-width: 900px">
+        <h4>Recent jobs</h4>
+        <em>Note: This table may be stale; refresh to see the latest state.</em>
+        <table class="table">
+          <tr>
+            <th>Job ID</th>
+            <th>Status</th>
+            <th>Time started</th>
+            <th>Time finished</th>
+            <th></th>
+          </tr>
+          {% for job in recent_job_data %}
+            <tr>
+              <td>{{job.id}}</td>
+              <td>{{job.status_code}}</td>
+              <td>{{job.human_readable_time_started}}</td>
+              <td>{{job.human_readable_time_finished}}</td>
+              <td>
+                <button ng-click="getJobOutput('{{job['id']}}')">
+                  View Output
+                </button>
+              </td>
+            </tr>
+          {% endfor %}
+        </table>
+        <div ng-if="showJobOutput">
+          <h4>Job Output</h4>
+          <p>
+            <[jobOutput]>
+          </p>
+        </div>
+      </md-card>
 
-    <ul>
-      {% for counter in counters %}
-        <li>
-          {{counter.description}} : {{counter.value}}
-        </li>
-      {% endfor %}
-    </ul>
+      <md-card class="oppia-page-card oppia-long-text" id="configcard" ng-if="currentTab === 'TAB_CONFIG'" style="max-width: 1050px">
+        <h3>Configuration properties</h3>
+        <div class="container" ng-if="isNonemptyObject(configProperties)" style="margin-left: 0;">
+          <div ng-repeat="(configPropertyId, configPropertyData) in configProperties">
+            <div class="row protractor-test-config-property" style="padding-bottom: 10px;">
+              <span class="col-lg-2 col-md-2 col-sm-2">
+                <em class="protractor-test-config-title"><[configPropertyData.description]></em>
+              </span>
+              <span class="col-lg-6 col-md-6 col-sm-6">
+                <schema-based-editor schema="configPropertyData.schema" local-value="configPropertyData.value">
+                </schema-based-editor>
+              </span>
+              <span class="col-lg-2 col-md-2 col-sm-2">
+                <button type="button" class="btn btn-default" ng-click="revertToDefaultConfigPropertyValue(configPropertyId)">Revert to default</button>
+              </span>
+            </div>
+          </div>
 
-    <h3>Search Index</h3>
+          <button ng-click="saveConfigProperties()" class="protractor-test-save-all-configs">Save</button>
+          <button ng-click="reloadConfigProperties()">Undo changes</button>
+        </div>
 
-    <button ng-click="clearSearchIndex()">Clear Search Index</button>
+        <div ng-if="!isNonemptyObject(configProperties)">
+          No configuration properties are available.
+        </div>
+      </md-card>
 
-    <h3>Topic Similarities</h3>
+      <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px;" ng-if="currentTab === 'TAB_MISC'">
```

Comment:
Here and elsewhere, refer instead to the constant (that I mentioned in an earlier comment).